lib/meta.js

```diff
@@ -195,7 +249,7 @@ async function generateAssetsAudio(storage) {
     exiftool = [
       "firejail",
       "--quiet", `--profile=${PROFILE}`, `--private=${p.dir}`,
-      EXIFTOOL, "-b", "-Picture", `./${p.base}`
+      EXIFTOOL, "-b", "-Picture", `${p.dir}/${p.base}`
```

Comment:
nope, this was correct, except your jailing is not working. Turn off jail in your config, and do not change this